devito/compiler.py

```diff
@@ -32,7 +45,20 @@ def __str__(self):
 
 
 class GNUCompiler(Compiler):
-    """Set of standard compiler flags for the GCC toolchain"""
+    """Set of standard compiler flags for the GCC toolchain
+
+    :param openmp: boolean indicating if openmp is enabled. False by default
```

Comment:
Do these need to be repeated in every child class? Does Sphinx not use the class-inheritance relationships?